api_app/serializers.py

```diff
@@ -921,9 +929,18 @@ def validate(self, attrs):
             # we are in an update
             return attrs
         if "organization" in attrs and attrs["organization"]:
-            if attrs.pop("organization").owner != attrs["owner"]:
+            org = attrs.pop("organization")
+            if not (
+                org.owner == attrs["owner"]
+                or (
+                    self.context["request"].user.has_membership()
+                    and self.context["request"].user.membership.organization.pk
+                    == org.pk
+                    and self.context["request"].user.membership.is_admin
+                )
+            ):
```

Comment:
Ok we starts to need some sort of comment in this boolean expression because it is breaking my mind.

This should mean
```

we raise ValidationError() when
1 - we are not owner  OR
2 - we are not admin of the same org
```
If I did not fucked up the boolean expression, the check seems valid to me, I would just suggest to add a comment because it took me 5 minutes to understand what was going on